client/client.go

```diff
@@ -163,3 +163,49 @@ func Redaqtionen(db *sqlx.DB, DFUNKT_URL string) func(c *gin.Context) {
 		})
 	}
 }
+
+// Page for individual members
+func Member(db *sqlx.DB, ds *DarkmodeStatus) func(c *gin.Context) {
+	return func(c *gin.Context) {
+		memberID := c.Param("member")
+
+		member, err := database.GetMember(db, memberID)
+		if err != nil {
+			c.Redirect(http.StatusNotFound, "")
```

Comment:
Shouldn't be a redirect if it's not a 3XX code (pretty sure gin will complain about this, but it's also against the web spec or whatever that thing is called)
Should probably be something `c.Error` or `c.SetStatusCode` or whatever (I have no idea what methods exist)